ui/admin/app/components/roles/role/actions/index.hbs

```diff
@@ -3,38 +3,31 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-{{!TODO Refactor page to use hds dropdown in ICU-13879}}
-<Rose::Dropdown
-  @text={{t 'actions.manage'}}
-  @dropdownRight={{true}}
-  as |dropdown|
->
+<Hds::Dropdown as |dd|>
+  <dd.ToggleButton @text={{t 'actions.manage'}} @color='secondary' />
   {{#if (can 'addPrincipals role' @model)}}
-    <dropdown.link
+    <dd.Interactive
+      @text={{t 'resources.role.principal.actions.add-principals'}}
       @route='scopes.scope.roles.role.add-principals'
       data-test-manage-dropdown-principals
-    >
-      {{t 'resources.role.principal.actions.add-principals'}}
-    </dropdown.link>
+    />
   {{/if}}
-
   {{#if (can 'setGrantScopes role' @model)}}
-    <dropdown.link
+    <dd.Interactive
+      @text='Manage Scopes'
```

Comment:
The `@text` attribute should be a translation, not a bare string. I would look at the `dd.Interactive` just above as an example. Note that it is self-closing as well.